data/meson.build

```diff
@@ -22,6 +22,10 @@ appstream_file = i18n.merge_file(
   install_dir: join_paths(get_option('datadir'), 'metainfo')
 )
 
+appstreamcli = find_program('appstreamcli', required: false, disabler: true)
```

Comment:
You need to replace the old call to `appstream-util` for this one, look a few lines below :)

Reply:
Oops, I missed that. Fixed.